src/js/views/maps/legend/LayerLegendView.js

```diff
@@ -43,27 +44,39 @@ define([
 
       /** @inheritdoc */
       render() {
+        const paletteType = this.model.get("paletteType");
+        if (!paletteType || paletteType === "categorical") {
```

Comment:
Can `paletteType` be undefined? Looks like it defaults to "categorical"

Reply:
ouioui you're right. removed check